docs/safety.md

```diff
@@ -0,0 +1,150 @@
+# Safety Policies & Procedures
+
+The importance of this safety guide is to:
+
+1. Ensure nobody gets harmed and reduce the risk of getting harmed
+1. How to prevent easily avoidable mistakes
+1. Reduce damage
+
+## General
+
+*If something starts smoking:*
+
+- Press the E-STOP and turn off all power immediately
+- Clear the lab
+- Call for help
+
+*In case of fire:*
+
+- There is a fire extinguisher on inside of the g-wagon
+- There is a larger fire extinguisher right outside 701
+
+*If something gets in the eye:*
+
+- There is an eye wash in each 701 sink
+- Do not rub eye
+- Call for help
+
+*If bleeding:*
+
+- Call for help
+- Apply pressure to the wound
+- First-aid kit is near the exit of 701
+- Wash hands
+- Use disinfectant
+- Use gauze/band-aids
+
+*If bleeding doesn't stop:*
+
+- Keep applying pressure
+- Go to urgent care
```

Comment:
An accident report will have to be filled out. Inform technician/instructor of wound